pages/linux/ikaros.md

```diff
@@ -0,0 +1,16 @@
+# ikaros
+
+> Vanilla OS Tool for managing drivers for your device.
+> More information: <https://github.com/Vanilla-OS/Ikaros>.
+
+- Install drivers for your device:
+
+`ikaros install {{device}}`
+
+- Automatically install drivers for your device:
+
+`ikaros auto-install {{device}}`
```

Comment:
Could you please clarify, what the difference between those two is? I don't feel it's obvious

Reply:
Ikaros is still in development so this behavior might change (will open a PR when it happens) , it is essentially a replacement for the `ubuntu-drivers-common` package for Debian and other distributions. 

In this case, the `install` command would ask for you to choose the Driver for your device (i.e. take NVIDIA drivers on Linux, for instance, there are tons of different drivers [proprietary and open source ones like Nouveau] so this allows users to choose their desired ones) and for the `auto-install` it installs the recommended driver for your device automatically instead of interactively asking to you.

Reply:
> In this case, the `install` subcommand would ask for you to choose the Driver for your device (i.e. take NVIDIA drivers on Linux, for instance, there are tons of different drivers [proprietary and open source ones like Nouveau] for the same GPU so this subcommand allows users to choose their desired ones) and for the `auto-install` it installs the recommended driver for your device automatically instead of interactively asking to you.

Sounds like you might mean that 😄:

```suggestion
- Interactively install drivers for your device:

`ikaros install {{device}}`

- Automatically install the recommended drivers for your device:

`ikaros auto-install {{device}}`
```